WcaOnRails/spec/models/user_spec.rb

```diff
@@ -65,16 +65,7 @@
     user.confirm
   end
 
-  it "allows demotion of a senior delegate with no subordinate delegates" do
-    senior_delegate = FactoryBot.create :senior_delegate
-
-    senior_delegate.delegate_status = ""
-    senior_delegate.region_id = nil
-    expect(senior_delegate.save).to eq true
-    expect(senior_delegate.reload.delegate_status).to eq nil
-  end
```

Comment:
Now demotion of senior delegates doesn't depend on subordinate delegates because delegates are not connected to senior delegates, instead are connected to regions.

Couldn't think of any replacement test as well.